src/nuxt/nuxt-wizard.ts

```diff
@@ -149,16 +153,32 @@ export async function runNuxtWizardWithTelemetry(
   await runPrettierIfInstalled();
 
   clack.outro(
-    buildOutroMessage(shouldCreateExamplePage, shouldCreateExampleButton),
+    buildOutroMessage(
+      shouldCreateExamplePage,
+      shouldCreateExampleButton,
+      deploymentPlatform,
+    ),
   );
 }
 
 function buildOutroMessage(
   shouldCreateExamplePage: boolean,
   shouldCreateExampleButton: boolean,
+  deploymentPlatform: DeploymentPlatform | symbol,
 ): string {
+  const canImportSentryServerConfigFile =
+    deploymentPlatform !== 'vercel' && deploymentPlatform !== 'netlify';
+
   let msg = chalk.green('\nSuccessfully installed the Sentry Nuxt SDK!');
 
+  if (canImportSentryServerConfigFile) {
+    msg += `\n\nAfter building your Nuxt app, you need to ${chalk.cyan(
+      '--import',
+    )} the Sentry server config file.\n\nFor more info see: ${chalk.cyan(
```

Comment:
l: Feel free to go with something else but I was a bit confused what I'd need to do at first. 

```suggestion
    )} the Sentry server config file when running your app.\n\nFor more info see: ${chalk.cyan(
```